src/database/DataTypes/IndicatorValue.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import Realm from 'realm';
+
+/**
+ * An abstract object which contains metadata describing an indicator row or column.
+ *
+ * @property  {string}              id
+ * @property  {string}              facilityId
+ * @property  {Period}              period
+ * @property  {IndicatorAttribute}  column
+ * @property  {IndicatorAttribute}  row
+ * @property  {string}              value
+ */
+export class IndicatorValue extends Realm.Object {}
+
+export default IndicatorValue;
+
+IndicatorValue.schema = {
+  name: 'IndicatorValue',
+  primaryKey: 'id',
+  properties: {
+    id: 'string',
+    facilityId: 'string',
```

Comment:
this isn't really needed since there will only ever be this stores values in the database. Also introduces terminology (Facility) which isn't used in the mobile code base which I for some reason really don't like 🤣 If we do keep it can we please just call it `storeId`?

Reply:
I agree, I'm not sure what the reasoning was for facility, but I'd like to change it on desktop, too. I'll change it here for now and piggy back the change to desktop when I next get a chance to update the structure.

Reply:
b7b32832.